hello-core-logic/listBoardMimicTrello.js

```diff
@@ -0,0 +1,30 @@
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill'],
+    },
+    'Cuisine': {
+      'Region':['African', 'European', 'Brazilian', 'Asian'] }
+};
+
+function listBoards(boardName) {
+  let dash = "\n------------------\n";
+  let result = dash;
+  let count = 0;
+  for(let boards in boardName) {
+    if (boardName !== '') {
```

Comment:
So, you named your input variable `boardName`, but it is not a string. You pass in the `hello` object. So this if-statement will always be true since an object will never be equal to the empty string.

Also, in a for-in loop, if the object is empty, it will not perform the loop, so there is no need to check if it is empty